Invalid usage of PlatformEntry interface

https://github.com/rabix/composer/blob/master/src/app/auth/model/auth-credentials.ts#L16
uses a `devTokenURL` in two locations which is not part of the `PlatformEntry` interface.

```
ERROR in src/app/auth/model/auth-credentials.ts(30,13): error TS2322: Type '{ "https://api.sbgenomics.com": { "name": string; "shortName": string; "platformURL": string; }; ...' is not assignable to type '{ [key: string]: PlatformEntry; }'.
  Property '"https://api.sb.biodatacatalyst.nhlbi.nih.gov"' is incompatible with index signature.
    Type '{ "name": string; "shortName": string; "devTokenURL": string; }' is not assignable to type 'PlatformEntry'.
      Object literal may only specify known properties, and '"devTokenURL"' does not exist in type 'PlatformEntry'.

```

--- a/src/app/auth/model/auth-credentials.ts
+++ b/src/app/auth/model/auth-credentials.ts
@@ -27,7 +27,7 @@ export class AuthCredentials implements UserPlatformIdentifier {
         "https://api.sb.biodatacatalyst.nhlbi.nih.gov": {
             "name": "BioData Catalyst",
             "shortName": "BDCatalyst",
-            "devTokenURL": "https://platform.sb.biodatacatalyst.nhlbi.nih.gov/developer/#token",
+            "platformURL": "https://platform.sb.biodatacatalyst.nhlbi.nih.gov/developer/#token",
         },
         "https://cgc-api.sbgenomics.com": {
             "name": "Cancer Genomics Cloud",
@@ -42,7 +42,7 @@ export class AuthCredentials implements UserPlatformIdentifier {
         "https://api.sevenbridges.cn": {
             "name": "Seven Bridges (China)",
             "shortName": "SBG-CN",
-            "devTokenURL": "https://platform.sevenbridges.cn/developer#token",
+            "platformURL": "https://platform.sevenbridges.cn/developer#token",
         },
     };
 

